### PR TITLE
Scraper Class

### PR DESCRIPTION
**Pull request by akashrajkn**
Created a `ForwardScraper` class with Spoofing.

- [x] Kickstarter scraper subclass
- [x] Ulule scraper subclass
- [x] Start some good scraper subclass

### CODE REVIEW

File: forward43/scraper_kickstarter.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        ''' Main Scraper function '''
+        for category in self.category_ids:
+            print('Category: {}'.format(category))
+            projects = []
+
+            for page in range(1, self.num_pages + 1):
+                print('    Page: {}'.format(page))
+
+                url       = self.get_url(category, page)
+                response  = self.get_response(url)
```
Comment (andrewsutjahjo) [inline forward43/scraper_kickstarter.py:59]: This could use a bit of error handling, otherwise one timeout will break the whole pipeline

**Review by andrewsutjahjo — CHANGES_REQUESTED** [review]
Nice one!
Good abstractions into classes, and good documentation. It just needs a bit of a cleanup of some unused code, maybe a logger instead of prints, and some error handling so nothing breaks from an errant spanner in the works.
A few simple unit tests would be good to ensure that anyone working on this code after you will know when they break it :)

**Review by andrewsutjahjo — APPROVED** [review]
Nice one!

**Comment by akashrajkn** [comment]
I will add unit tests in a separate PR.

**Pull request merged**